src/snowflake/snowpark/modin/plugin/_internal/ordered_dataframe.py

```diff
@@ -1052,6 +1053,9 @@ def join(
         right: "OrderedDataFrame",
         left_on_cols: Optional[list[str]] = None,
         right_on_cols: Optional[list[str]] = None,
+        left_match_col: Optional[str] = None,
+        right_match_col: Optional[str] = None,
+        match_comparator: Optional[Enum] = None,
```

Comment:
Can't type as MatchComparator due to circular import issue

Reply:
you will be able to use MatchComparator with quote, like "MatchComparator" here, right?